CorsixTH/Lua/dialogs/menu.lua

```diff
@@ -540,6 +540,76 @@ local function hotkey_value_label(hotkey_name, hotkeys)
   return string.upper(array_join(hotkey_value, "+"))
 end
 
+local function audio_options(self, app, menu)
+  if not app.audio.has_bg_music then return end
+  local hotkeys = app.hotkeys
```

Comment:
- Add an empty line above this line. 

  The first line handles something else than this and the next lines. If you add an empty line between such blocks, you can find closely related code for some part more easily.

  Also, it helps in navigating. When looking for some specific part, you can start reading code and when you realize it's not what you are looking for, you can simply skip to code below the next empty line. You don't need to carefully keep reading to find the spot where the code starts doing something else.

  One step further, if you would start each such block with a single line of comment that describes what the code block is doing, you can navigate even quicker by reading the comment line, decide if the code block is what you're looking for, and if not directly skip to the next block with the next comment line above it.

- You don't seem to need `hotkeys` until further down in the function, so move that line closer to first use.